lightning/src/ln/channel.rs

```diff
@@ -11283,6 +11293,7 @@ impl<'a, 'b, 'c, ES: Deref, SP: Deref> ReadableArgs<(&'a ES, &'b SP, &'c Channel
 			(54, pending_funding, optional_vec), // Added in 0.2
 			(55, removed_htlc_failure_attribution_data, optional_vec),
 			(57, holding_cell_failure_attribution_data, optional_vec),
+			(58, interactive_tx_signing_session, option),
```

Comment:
Could you added the trailing comment here, too? Not sure why we aren't consistently doing this for other fields.